shiny/_main.py

```diff
@@ -399,6 +404,17 @@ def run_app(
 
     maybe_setup_rsw_proxying(log_config)
 
+    if is_workbench() and kwargs.get("ws_per_message_deflate"):
+        # Workaround for nginx/uvicorn issue within Workbench
+        # https://github.com/rstudio/rstudio-pro/issues/7368#issuecomment-2918016088
+        warnings.warn(
+            "Overwriting kwarg 'ws_per_message_deflate'=True to False to avoid breaking issue in Workbench",
+            stacklevel=2,
+        )
+        kwargs["ws_per_message_deflate"] = False
+    elif is_workbench():
+        kwargs["ws_per_message_deflate"] = False
```

Comment:
```suggestion
    if is_workbench():
        if kwargs.get("ws_per_message_deflate"):
            # Workaround for nginx/uvicorn issue within Workbench
            # https://github.com/rstudio/rstudio-pro/issues/7368#issuecomment-2918016088
            warnings.warn(
                "Overwriting kwarg `ws_per_message_deflate=True` to `False` to avoid breaking issue in Workbench",
                stacklevel=2,
            )
        kwargs["ws_per_message_deflate"] = False
```